apertium-crh.crh.lexc

```diff
@@ -18,107 +23,119 @@ Multichar_Symbols
 %<adv%>           ! Adverb                      !
 %<v%>             ! Verb                        !
 %<cnjcoo%>        ! Coordinating conjunction    !
-%<cnjsub%>        ! "subordinating conjunction" (ki)
-%<cnjadv%>        ! 
-%<qst%>           ! Question word
-%<sym%>
+%<cnjsub%>        ! Subordinating conjunction (ki)
+%<cnjadv%>        ! Adverbial conjunction
+%<qst%>           ! Modal question particle     !# м{I}
+                  
+%<sym%>           ! Symbol
 %<cop%>           ! Copula
-%<postadv%>       ! 
-%<det%>           ! 
-%<num%>           ! 
+%<postadv%>       ! Post of Adverb
+%<det%>           ! Determiner
+%<num%>           ! Numeral
```

Comment:
Why are these words highlighted?

Reply:
the highlighting is due to `git diff` recognizing that only part of a line has been changed

Reply:
Can I remove this highlighting and if yes how?

Reply:
That highlighting is solely a matter of the interface on which you are viewing the changes. I am not aware of any way to turn it off.

Reply:
Ok thanks anyways